### PR TITLE
Update gameplay UI

### PR DESCRIPTION
**Pull request by keeganhauser**
Redid some of the logic of displaying powerups. Whitespace is looking funky in GitHub again, so don't mind that lol

### CODE REVIEW

File: TKGame/Items/Potion/Potion Components/C_PotionItem_Physics.cs [hunk 1]
```diff
@@ -23,8 +25,22 @@ void IPhysicsComponent.Update(Entity entity, GameTime gameTime/*, World &world*/
...
+                        case 1:
+					        Player.Instance.PickUpPowerUp(new C_Fire_SpecialAttack());
+                            break;
+                        case 2:
+					        Player.Instance.PickUpPowerUp(new C_Fire_UltimateAttack());
+                            break;
+                        default:
+                            break;
+                    }
+				}
```
Comment (aaronw84) [inline TKGame/Items/Potion/Potion Components/C_PotionItem_Physics.cs:43]: Nice

**Review by aaronw84 — APPROVED** [review]
Good

File: TKGame/UI/GameMenu.cs [hunk 2]
```diff
@@ -208,26 +214,44 @@ private void AddWidgetToHeaderPanel(Widget widget, string text = null, bool isVi
...
             // TODO: Make this more flexible for when other powerups are collected
-            foreach (Widget widget in panelHsp.Widgets.Where(w => !w.Visible))
-                widget.Visible = true;
+            foreach (IAttackComponent attackComponent in attackComponents) 
+            {
+                if      (attackComponent is C_Fire_SpecialAttack)      ShowWidget("Q");
+				else if (attackComponent is C_Fire_UltimateAttack)     ShowWidget("E");
+				else if (attackComponent is C_Fire_MovementAttack)     ShowWidget("Shift");
+                else if (attackComponent is C_Ice_PrimaryAttack)       ShowWidget("I");
+                else if (attackComponent is C_FireStone_PrimaryAttack) ShowWidget("O");
```
Comment (NateCastleOIT) [inline TKGame/UI/GameMenu.cs:248]: I'm guessing this will change with aaron's change

**Review by NateCastleOIT — APPROVED** [review]
y

**Pull request merged**